description: Add structure for Lectures teaching stage (#10)
open-education-hub/methodology: description.md

This file contains all the topics that are covered by the Open Education Hub methodology.
Each topic in this file contains a small description and links to the files cover it more thouroughly.

## Teaching Stages

### Lectures

Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.
Interaction is (quite obviously) important, but it won't scale to a larger audience and, because the entire communication medium is shared, is limited to on conversation only.
Labs, for example, allow each participant to focus on one item they could work on.

> Seminars are close to lectures, except that they typically involve a smaller group of people and are naturally interactive, with the relationship being less asymmetrical.

Reviewer: I think this build-up does not lead anywhere, consider talking about labs and seminars to introduce the need for materials (in multiple forms) to make lectures as effective.

Reviewer 2: On a second thought, you could just rephrase this to: "Lectures are different from practical sessions (link to practical sessions) by ...".

Author: @Alex-deVis , @RazvanN7 , indeed. I rephrased this.

Materials are required for delivering lectures.
In the following sections, we detail steps in preparing and (re)using lecture materials in Open Education Hub.

We organize lecture materials according lecture-specific activities:

* **Visual support materials** are used as support material for giving talks and engaging participants in discussions.
  They consist of:
  * **Slides** are the typical structured and simplified format for organizing ideas.

>     The lecturer typically walks through slides and presents information by expanding ideas on the slides, and by engaging in discussions based on those ideas.

Reviewer: ```suggestion
    The lecturer walks through slides and presents information by expanding ideas on the slides, and by engaging in discussions based on those ideas.
```
`typical` is repeated.

>   * **Media files** are generally images or videos that represent a concept in an easier to grasp format.

Reviewer 2: I don't think this category is necessary. You can say that media and reading are embedded into slides.

Author: Maybe some media may not be embedded. I think I can leave it separate, but mention that media is typically embedded into slides.

### Lectures

Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.

> Interaction is (quite obviously) important, but it won't scale to a larger audience and, because the entire communication medium is shared, is limited to on conversation only.

Author: Elaborate on the last sentence being wrong?

> Interaction is (quite obviously) important, but it won't scale to a larger audience and, because the entire communication medium is shared, is limited to on conversation only.

Reviewer 3: > is limited to on conversation only

What is "on conversation"? It doesn't sound right. Reading it again, it seems it should be without "on": "is limited to conversation only"

Labs, for example, allow each participant to focus on one item they could work on.
Seminars are close to lectures, except that they typically involve a smaller group of people and are naturally interactive, with the relationship being less asymmetrical.

#### Lecture Goals

The ideal lecture transfers the greates amount of knowledge / information from lecturer to participants.
The lecture goals are aligned to that.
These should be:
- getting participants to attend
- getting participants to assimilate information
- getting participants to be (inter)active
- providing high-quality material before, during and after lecture
- correlating with other stages for concerted knowledge transfer
- exciting participants about the topic
- building a community of participants around the topic

#### Lecture Metrics

- attendance

> - interaction

Author: You can measure interaction why the amount of questions you get, students who take part in it.

- exam grades
- participant feedback
- content validation / approval by participants and / or outsiders

#### Lecture Materials

- types of materials
- focus on interactivity
- availability outside of class
- dynamic
- delivery / online materials vs offline materials

#### Lecture Delivery & Structure

- hooking participants in: relevance of topic, connection to other topics
- interaction
- dynamic content, building it step by step
- quizzes throughout lecture: mass validation of understanding
- rough high-level structure to be adapted to each lecturer's personality and view of things
- conclusion
- layered approach: top-to-bottom

#### Connection to Other Stages

- lecture before lab, seminar
- demos during both lectures and labs
- direct references between lectures and labs (forward reference from lectures to labs)

> - direct references between assignments and lectures

Reviewer 3: Add that assignments should build on top of lectures and labs.

Author: for sure

#### Best Practices

- layered approach
- real-world use cases
- personal lecturer experiences & anecdotes
- lecturer to be part of projects & activities related to the topic
- materials to be OER
- encourage contributions
- quiz at the beginning based on earlier course

> - some feedback form (not to often, because it will bore participants)

Reviewer 3: ```suggestion
- some feedback form (not too often, because it will bore participants)
```

- record sessions, apply feedback, reiterate content

> - have demos

Reviewer 3: ```suggestion
- have demos
- use the blackboard, draw figures on the spot
- avoid slides with images, definitely avoid slides with text
```

### Lectures

Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.
Interactivity is important but is limited by the larger audience: not all participants will be able to actively engage in conversations;

> however, items that don't require conversation, such as quizzes, are a good way to engage all participants, as we discuss below.

Reviewer: ```suggestion
however, items that don't require conversation, such as [quizzes](#quizzes), are a good way to engage all participants, as we discuss below.
```

Practical sessions, on the other hand, put participants in the position where they work on concrete items, and get involved in smaller groups discussions, with a less asymmetrical interaction.

Materials are required for delivering lectures.
In the following sections, we detail steps in preparing and (re)using lecture materials in Open Education Hub.

#### Lecture Materials

We organize lecture materials according to lecture-specific activities.
We reference them as presented in the ["Types of Content" section](#types-of-content).

* **Visual support materials** are used as support material for giving talks and engaging participants in discussions.

>   They consist of:

Reviewer: ```suggestion

  They consist of:
```
Without an empty line before the indented line, there is no change in the view. Is this for stylistic reasons only?

Author: @Alex-deVis , I don't understand this.

  * [**Slides**](#slides) are the typical structured and simplified format for organizing ideas.

>     The lecturer walks through slides and presents information by expanding ideas on the slides, and by engaging in discussions based on those ideas.

Reviewer 3: ```suggestion
    The lecturer may walk through slides and present information by expanding ideas on the slides, and by engaging in discussions based on those ideas.
```

  * [**Media files**](#media) are generally images or videos that represent a concept in an easier to grasp format.
    Diagrams, photos, recordings of actions / processes, recorded simulations fall under this category.
    They can be used either stand-alone or embedded into slides.

* **Practical support materials** are used to demonstrate actual real-world applications of presented concepts.
  The lecturer is the one who uses the practical support materials to do live demonstrations and highlight concrete use cases.
  This may not be possible for certain topics, either because they are highly theoretical or because it is difficult to have a live setup.
  Where a live demonstration is not possible, a video recording (as a visual support material) may be used.
  They consist of [**Demos**](#demos).

> * **Assessment materials** are used to evaluate knowledge and skills of participants, and to provide feedback on spots they need to focus more on.

Reviewer 3: ```suggestion
* **Assessment materials** are used to evaluate the knowledge and skills of participants, and to provide feedback on spots they need to focus more on.
```

  Typically they are to be used live during lectures, with their results providing immediate feedback and triggering discussions.
  They consist of [**Quizzes**](#quizzes).

* **Offline self-study materials** are to be used by participants outside lectures.
  They are detailed in the ["Self-Study" section](#self-study).

As other types of content, lecture materials are publicly available, and also open to contributions.

> Lecture materials are meant to be used live by teachers during lectures and then offline by learners, to improve gained knowledge and skills.

Reviewer: ```suggestion
Lecture materials are meant to be used live by educators during lectures and then offline by learners, to improve gained knowledge and skills.
```

Visual support materials are directly available on the course website;
the other materials are available either as downloadable archives or directly in the course repository.

#### Lecture Content Structure

Teachers have individual styles and preferences.
Hence, they would define their prefered order of presenting topics and decide on the weight of each topic.

Even so, it is recommended that the types of materials shown above be intertwined during an actual lecture.
Slides are good for presenting general topics, but tend to become tern and difficult to follow.
As such, spreading demos and quizzes between slides is great to engage the audience and also provide a practical overview of items and an insight into topics that deserve continued attention.

One possibility is to have lectures focused primarily on demos and on quizzes and slides to be used sparringly, only as topic highlights.

In the ideal case, each topic in a lecture will be a balanced mix of demos, quizzes, slides and media file.
The dedicated teacher will be careful to adjust the types of content to the topic being presented and to the audience.

> 

Reviewer 3: You usually ditch slides altogether and only use demos and quizzes. You should present this approach as well. Maybe split them into subsections? Although the heading would be 5 `#`s, so it would probably be too much.

Either way, I would emphasize experimenting with various mixes according to the type of audience. Maybe people from different cultures react differently to "stimuli" such as quizzes or demos.

Author: Agree. Will augment that.

#### Evaluating Lecture Content

It is difficult to evaluate lecture content without including the teacher delivery method itself.
A charismatic teacher will likely get good student feedback even with low-quality lecture content.

Even so, there are several metrics that we recommend to be considered:

> * Good quiz grades (during lectures) are generally a sign that the respective topics are well presented.

Reviewer: ```suggestion
- Good quiz grades (during lectures) are generally a sign that the respective topics are well presented.
```
We should decide on a way to present lists; other PRs use **-**.

> * Good assignment grades and fewer technical assignment clarfications mean that the topics for assignments were well covered during lectures.

Reviewer 3: ```suggestion
* Good assignment grades and fewer technical assignment clarifications requested by students mean that the topics for assignments were well covered during lectures.
```

> * Website accesses for particulare materials signal the topic is deemed well presented by students (and others - as it is public content).

Reviewer 3: ```suggestion
* Website accesses for particular materials signal the topic is deemed well presented by students (and others - as it is public content).
```
Or rather that it's of interest to them, possibly even badly presented. If something goes smoothly and is clear, why would you revisit it?
